src/nimvscode/nimSuggestExec.nim

```diff
@@ -154,7 +154,7 @@ proc closeNimsuggestProcess*(file: cstring) {.async.} =
       console.error("closeCachedProcess failed", getCurrentException())
     finally:
       nimSuggestProcessCache[file] = jsUndefined.to(Promise[NimSuggestProcessDescription])
-      discard jsDelete nimSuggestProcessCache[file]
+      discard delete(nimSuggestProcessCache, file)
```

Comment:
```suggestion
      discard nimSuggestProcessCache.delete(file)
```

There is no return value in my import of it but there should be one with `bool` -- that can be fixed later.

The `discard` is still good to keep for that reason.